ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -72,13 +94,47 @@ const triggerSet: TriggerSet<Data> = {
       id: 'R6S Sticky Mousse',
       type: 'StartsUsing',
       netRegex: { id: 'A695', source: 'Sugar Riot', capture: false },
-      response: Responses.spread(),
+      response: Responses.spreadThenStack(),
+    },
+    {
+      id: 'R6S Color Riot Debuff Tracker',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['1163', '1164'], capture: true },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => data.colorRiotTint = matches.effectId === '1163' ? 'warm' : 'cool',
     },
     {
       id: 'R6S Color Riot',
       type: 'StartsUsing',
-      netRegex: { id: ['A691', 'A692'], source: 'Sugar Riot' },
-      response: Responses.tankCleave(),
+      netRegex: { id: ['A691', 'A692'], source: 'Sugar Riot', capture: true },
+      alertText: (data, matches, output) => {
+        if (data.role !== 'tank')
+          return output.avoidCleave!();
+
+        const coolInOut = matches.id === 'A691' ? output.in!() : output.out!();
+        const warmInOut = matches.id === 'A692' ? output.in!() : output.out!();
+
+        switch (data.colorRiotTint) {
+          case 'warm':
+            return output.coolCleave!({ dir: coolInOut });
+          case 'cool':
+            return output.warmCleave!({ dir: warmInOut });
+          default:
+            return output.tankCleave!();
```

Comment:
No strong feelings necessarily, but I wonder whether this might be worth a custom "avoid cleave -- be on boss ring". Otherwise I can see people autopiloting out.

Reply:
how does `Be on boss hitbox (avoid tank cleaves)` sound?